packages/server-wallet/server-wallet.api.md

```diff
@@ -76,6 +76,26 @@ export type DatabasePoolConfiguration = {
     min?: number;
 };
 
+// @public (undocumented)
+export class DBAdmin {
```

Comment:
I did add documentation to the methods. @geoknee is there anything special I need to do?

Reply:
Thanks! I wouldn't worry about this too much now, it should show up as documented when my open PR lands.

Reply:
This one https://github.com/statechannels/statechannels/pull/3179